core/forkid/forkid_test.go

```diff
@@ -328,7 +328,7 @@ func TestValidation(t *testing.T) {
 
 		// Local is mainnet Gray Glacier, and isn't aware of more forks. Remote announces Gray Glacier +
 		// 0xffffffff. Local needs software update, reject.
-		{&timestampedConfig, 15050000, 0, ID{Hash: checksumToBytes(checksumUpdate(0xf0afd0e3, math.MaxUint64)), Next: 0}, ErrLocalIncompatibleOrStale},
+		{params.MainnetChainConfig, 15050000, 0, ID{Hash: checksumToBytes(checksumUpdate(0xf0afd0e3, math.MaxUint64)), Next: 0}, ErrLocalIncompatibleOrStale},
```

Comment:
I think this will turn stale once we move MainnetConfig to also be aware of Shanghai

Reply:
Yes, but this problem potentially also exists in other test cases here.

Reply:
Let me know if you have a cool idea for a solutiojn.